src/pages/search/advanced_search/FileStatusSelect.tsx

```diff
@@ -0,0 +1,53 @@
+import { MenuItem, Select, SelectChangeEvent } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam } from '../../../api/searchApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+
+enum FileStatus {
+  hasPublicFiles = 'hasPublicFiles',
+  noFiles = 'noFiles',
+  showAll = 'showAll',
```

Comment:
I stedet for å legge inn en ny verdi for å støtte dette kunne vi hatt `<Select displayEmpty (...)>` også, og default verdi hadde vært tomt. Personlig liker jeg nok det best, siden de to andre verdiene her må stemme overens med backend, mens denne er noe vi har funnet på bare for frontend, i tillegg til at det virker litt mystisk med at de to andre kommer i URLen, mens denne ikke gjør det.
Men det er mer en smaksak, så gjør det du selv syns er mest logisk :)

Reply:
Synes det ser litt rart ut når det er det eneste feltet uten default tekst i seg, og det fir mening for meg at det står "Vis alle" om man vil nullstille parameteret.

Reply:
Tenker at det ser litt rart ut når det er det eneste feltet uten noen form for takst, så valgte å beholde "Vis alle" 😬

Reply:
Mener ikke å ta vekk "Vis alle" nei, men å ev ta vekk den fiktive verdien `showAll`, og vise "Vis alle" når feltet ikke har noen verdi